### ISSUE TITLE
Will the tutorial be updated to 1.0 version

### ISSUE THREAD

**Issue by andrewfan** [mulesoft-labs/raml-tutorial-200#7]

**Comment by sichvoge** [comment]
Yes it will be. See in our forum [here](http://forums.raml.org/t/sneak-peek-raml-org-tutorial-whats-gonna-happen/1425/1) and the new repository that includes all the content not only the examples [here](https://github.com/raml-org/raml-tutorial).
